### DISCUSSION TITLE
Test Discussion

### DISCUSSION THREAD

**Discussion by vigneshbalu96** [Cyfrin/foundry-full-course-cu#1095] [General]
Hello, My fellow Blockchaineers! Looking forward to discuss with you all : )

**Comment by 0xusmanf** [comment]
Welcome onboard! LFG 🚀🔥

**Comment by koushhik** [comment]
❤️‍🔥 Hello, there glad to meet you guys and make your acquaintance.
so you guys are good with solidity..? 💻

**Comment by christo4-ayodele** [comment]
Hello, everyone, i am new to solidity and blockchain, looking forward to making more progress
